plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHiveMetadataListing.java

```diff
@@ -106,6 +106,45 @@ public class TestHiveMetadataListing
             Optional.of("SELECT 1"),
             OptionalLong.empty());
 
+    private static final Table CORRECT_TABLE = new Table(
+            DATABASE_NAME,
+            "correct_table",
+            Optional.of("owner"),
+            "table_type",
```

Comment:
nit: The constant name ends with `TABLE`. I would recommend using `MANAGED_TABLE` instead. Same for others.